src/parser.cpp

```diff
@@ -357,6 +357,8 @@ Variant* VariantParser<DoCheck>::parse(Variant* v) {
     parse_attribute("mandatoryPiecePromotion", v->mandatoryPiecePromotion);
     parse_attribute("pieceDemotion", v->pieceDemotion);
     parse_attribute("blastOnCapture", v->blastOnCapture);
+    parse_attribute("blastImmuneTypes", v->blastImmuneTypes, v->pieceToChar);
```

Comment:
Since `blastImmuneTypes` is not supported in combination with pseudo-royalty yet, it would be good to add a warning in the consistency checks below to make this explicit.